packages/kbn-optimizer/limits.yml

```diff
@@ -42,6 +41,7 @@ pageLoadAssetSize:
   embeddable: 87309
   embeddableEnhanced: 22107
   enterpriseSearch: 66810
+  entityManager: 17175
```

Comment:
what are these numbers? how come it's different from the old value?

Reply:
this sets a size limit for the frontend bundle, any build that increases the size of the bundle over the limit would fail. It's lower than the previous bundle because 1. I've removed code 2. looks like the previous value was manually set to give some buffer while this one was generated with a script